src/collegetown.py

```diff
@@ -0,0 +1,35 @@
+from os import environ
+from time import sleep
+
+from yelpapi import YelpAPI
+
+from src.constants import (
+    YELP_LATITUDE,
+    YELP_LONGITUDE,
+    YELP_RADIUS,
+    YELP_NUMBER_LIMIT,
+    YELP_QUERY_DELAY,
+)
+
+API_KEY = environ.get('YELP_API_KEY')
+
+yelp_api = YelpAPI(API_KEY)
```

Comment:
why not just initialize it with `environ.get()` here, if it's not used anywhere else?

Reply:
I don't get what you mean

Reply:
ohh jk lol